### PR TITLE
feat(material-experimental): option to filter tab-group harness by selected tab label

### PR DESCRIPTION
**Pull request by devversion**
Follow-up that addresses feedback from the original tab-group
harness PR. See: https://github.com/angular/components/pull/16728#discussion_r314819393

### CODE REVIEW

File: src/material-experimental/mdc-tabs/harness/tab-group-harness-filters.ts [hunk 1]
```diff
@@ -6,4 +6,6 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-export type TabGroupHarnessFilters = {};
+export type TabGroupHarnessFilters = {
```
Comment (crisbeto) [inline src/material-experimental/mdc-tabs/harness/tab-group-harness-filters.ts]: Should this have a doc string?

File: src/material-experimental/mdc-tabs/harness/tab-group-harness.spec.ts [hunk 2]
```diff
@@ -43,6 +43,13 @@ function runTests() {
     expect(tabGroups.length).toBe(1);
   });
 
+  it('should load harness for tab-group with selected tab label', async () => {
```
Comment (crisbeto) [inline src/material-experimental/mdc-tabs/harness/tab-group-harness.spec.ts]: Do we want a test for passing in a regex?

File: src/material-experimental/mdc-tabs/harness/tab-group-harness-filters.ts (hunk 1 above)
Comment (devversion) [reply src/material-experimental/mdc-tabs/harness/tab-group-harness-filters.ts]: Not sure. The boilerplate and other harnesses don't have a doc string for these either.

File: src/material-experimental/mdc-tabs/harness/tab-group-harness.spec.ts (hunk 2 above)
Comment (devversion) [reply src/material-experimental/mdc-tabs/harness/tab-group-harness.spec.ts]: done.

**Pull request merged**